adopt-a-dog/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Adaptadog</title>
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Francois+One%7CJacques+Francois" rel="stylesheet">
+</head>
+  <body>
+      <!-- <header> -->
+      <div class="container">
+        <img src="images/christopher-harris-57366-unsplash.jpg" alt="header image" height="700">
+        <button class="btn">Check Out</button>
+      </div>
+      <img src="images/cart-icon.svg" alt="shopping cart icon" height="40">
+      <h1>Adoptadog</h1>
+      <h2>Seattle's Premier Dog Adoption Agency</h2>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">/Dogs</a></li>
+          <li><a href="blog.html">/Blog</a></li>
+        </ul>
+      </nav>
+      <img src="images/eric-ward-610868-unsplash.jpg" alt="header image" height="250">
+      <h2>Adopt A Dog</h2>
+      <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus
+    am aut amet ant fugiatum, utem non reptat ad magnienecto volorent fugia nonsedi gendand itatquos as magnimpor aut molor autatia ad est officimus accae et omnia cus, od qui ate</p>
+    <!-- Murphy -->
+    <article class="container">
+      <img src="images/alex-lacayo-756508-unsplash.jpg" alt="murphy profile" height="180">
+      <h3>Murphy</h3>
+      <p><b>Cost to Adopt:</b>$123.45</p>
+      <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+      <button class="btn">Adopt</button>
+    </article>
+      <!-- Poppy -->
+    <div>
```

Comment:
a little inconsistent...you should use the the same parent container for each dog.